src/api/json_models/chat_completion.rs

```diff
@@ -10,7 +10,15 @@ pub struct Choice {
     pub message: Message,
 }
 
+#[derive(Deserialize, Clone)]
```

Comment:
See comment at [`src/api/chat_completions.rs`](src/api/chat_completions.rs). The `Clone` trait is not needed here.